pkg/controller/cloudmetrics/cloudmetrics_controller.go

```diff
@@ -293,3 +344,22 @@ func registerGaugeVectorMetrics(logger *logrus.Entry) {
 		customMetrics.Registry.MustRegister(metric.GaugeVec)
 	}
 }
+
+// func labelScrapedMetrics add labels to scraped metrics
+func (r *ReconcileCloudMetrics) labelScrapeMetrics( gaugeMetrics []CroGaugeMetric, scrapedMetrics []*providers.GenericCloudMetric) {
```

Comment:
small nit pick on this, could we rename this to something like `setGaugeMetrics` as this is what we are doing here, setting a value on an exposed metric with labels.

Reply:
sure